diff --git a/node_modules/html-to-image/lib/embedWebFonts.js b/node_modules/html-to-image/lib/embedWebFonts.js
index b1e4b44..f0dc5bd 100644
--- a/node_modules/html-to-image/lib/embedWebFonts.js
+++ b/node_modules/html-to-image/lib/embedWebFonts.js
@@ -138,7 +138,9 @@ function getCSSRules(styleSheets) {
             deferreds = [];
             // First loop inlines imports
             styleSheets.forEach(function (sheet) {
-                if ('cssRules' in sheet) {
+                if (sheet.hasOwnProperty('cssRules'))
+                // if ('cssRules' in sheet)
+                 {
                     try {
                         (0, util_1.toArray)(sheet.cssRules).forEach(function (item, index) {
                             if (item.type === CSSRule.IMPORT_RULE) {
